ts/src/test/Exchange/base/test.sharedMethods.ts

```diff
@@ -366,14 +368,14 @@ function removeProxyOptions (exchange: Exchange, skippedProperties: string[]) {
     return [ proxyUrl, httpProxy, httpsProxy, socksProxy ];
 }
 
-function setProxyOptions (exchange: Exchange, skippedProperties: string[], proxyUrl, httpProxy, httpsProxy, socksProxy) {
+function setProxyOptions (exchange: Exchange, skippedProperties: string[], proxyUrl: string, httpProxy: string, httpsProxy: string, socksProxy: string) {
     exchange.proxyUrl = proxyUrl;
     exchange.httpProxy = httpProxy;
     exchange.httpsProxy = httpsProxy;
     exchange.socksProxy = socksProxy;
 }
 
```

Comment:
@samgermain as @ttodua noted in a different PR skippedProperties shouldn't be typed like string[]

Reply:
> @samgermain as @ttodua noted in a different PR skippedProperties shouldn't be typed like string[]

do you have a link?
are you referring to `skippedProperties` specifically?